### DISCUSSION TITLE
3D Models Now Available!

### DISCUSSION THREAD

**Discussion by DipFlip** [limbo-ml/user#5] [Q&A] [answered]
Hi! 

Very cool project! 

Are any of the 3D models/environments be available? The environments are just 360 photos/HDRIs and not actual 3D scenes, right?

Thanks!

**Comment by tshead** [comment]
@DipFlip - thanks for your interest!  Roughly 2/3rds of the synthetic images use HDRI backgrounds.  The remainder use a true 3D scene of a refinery that provided a good industrial environment representative of nuclear-fuel-related facilities.

We hadn't contemplated sharing 3D models, but would be willing to consider it if you have a project in mind.  Keep in mind that only the UF6 cylinder models are ours to share, the distractor models, humans, and refinery environment were purchased from TurboSquid and can't be distributed further (but you could buy your own copies).

Cheers,
Tim

**Comment by DipFlip** [comment]
I see! We're working on simulations for autonomous container surveys using quadruped robots and drones. We're most interested in [campaign 20](https://limbo-ml.readthedocs.io/en/latest/campaign20.html). Is that one from 3D models and if so, could you link them for us to purchase too?

The UF6 models would be nice too if it's possible. We've just used cylinders for now.

Thanks again!

**Comment by tshead2** [comment]
@DipFlip - yes, campaign 20 is the 3D environment.  We'll put together a list of links to resources in the documentation and notify you here when it's done.  Releasing the original 3D models will take longer, but we can get that process started.

Cheers,
Tim

**Reply by DipFlip** [reply]
Really appreciate it! Thanks

**Comment by DipFlip** [comment]
@tshead any updates on this? Thanks!

**Reply by tshead2** [reply]
We're waiting for this to go through our review process.  Shouldn't be long.

Cheers,
Tim

**Reply by tshead2** [reply]
OK, it's taking longer than I would like, but is on track for the beginning of next month ...

**Reply by DipFlip** [reply]
Thanks for the update 👍!

**Reply by tshead2** [reply, marked as answer]
Well, that took too long.

But it's done! You can read about the new 3D model and texture downloads under [3D Models](https://limbo-ml.readthedocs.io/en/latest/models.html), and download the data using the same process as the rest of the Limbo images.

Cheers,
Tim

**Reply by DipFlip** [reply]
Thanks a lot @tshead2, appreciate it!
